jina/types/arrays/document.py

```diff
@@ -83,7 +83,17 @@ def __init__(self, docs: Optional[DocumentArraySourceType] = None):
                     raise ValueError(
                         f'DocumentArray got an unexpected input {type(docs)}'
                     )
-        self._update_id_to_index_map()
+        self._id_to_index = None
+
+    @property
+    def id_to_index(self) -> Dict:
```

Comment:
make it provate

Reply:
it's used in jina/types/document/graph.py so it can't be private (IDE used to show a warning)

Reply:
lets still make it private ignore graphDoc for now. graphDoc has been removed from our docs and will highly likely to be deprecated in 3.0

Reply:
changed